clock.py

```diff
@@ -18,7 +19,8 @@ def access_stack_overflow_api():
     if stack_exchange_api.have_logged_in(12) is False:
         message = "You haven't logged in for at least " + str(delta_hours) + " hours! \n " + \
                   "Access stackoverflow.com to save your login streak"
-        print("ERROR!\n" + message)
+        logging.basicConfig(level=logging.ERROR)
```

Comment:
Based on the statement here https://realpython.com/python-logging/, the library should log, by default, everything above WARNING. So for this particular error message, the `basicConfig` could be omitted, because it will log the ERROR message

> ... This is because, by default, the logging module logs the messages with a severity level of WARNING or above.